templates/telemetry/pre-account/pre-account.4.schema.json

```diff
@@ -109,7 +109,13 @@
           "type": "number"
         },
         "scalars": {
-          "type": "object"
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "additionalProperties": {
+              "type": "integer"
```

Comment:
can't they be integer, boolean or string?  You may want to include https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/scalars.1.schema.json  using @TELEMETRY_SCALARS_1_JSON@